docs/application/dotnet/guides/opentk/window-of-opentk.md

```diff
@@ -0,0 +1,121 @@
+# Window of OpenTK
+
+In `OpenTK` application, `OpenTKGameApplication` creates a `Window` after calling `OnCreate` method. The `Window` is provided
+as a property of `TizenGameApplication`. This document will describe how to use this `Window` in your `OpenTK` application.
+
+`Window` defines many events, including: [input event](#input-event), [window related](#window-related-event), 
+[render related event](#render-related-event), you can add event handlers on those event if you need.
+
+## Input Event
+
+In some app scenario, you want to do some actions while an app user input something. To do this, you can add event hander on the input event of `Window`. Also, you can use `key` and `mouse` event through using `Window` of `OpenTK` if a device supports both of them.
+
+```C#
+private IGameWindow mainWindow;
+
+protected override void OnCreate()
+{
+    mainWindow = Window;
+    
+    mainWindow.MouseMove += MouseMoveEventHandler;    // add event handler on mouse move event
+    mainWindow.KeyDown += KeyDownEventHandler;        // add event handler on key down event    
+}
+
+private void MouseMoveEventHandler(object sender, MouseMoveEventArgs e)
+{
+    if (e.Mouse[MouseButton.Left])      // get mouse move event which happening with click left button of mouse
+    {
+        float x = (float)(e.XDelta);    // the move distance in x-axis
+        float y = (float)(e.YDelta);    // the move distance in y-axis
+        // do your action
+    }
+    
+    if (e.Mouse[MouseButton.Right])     // get mouse move event which happening with click right button of mouse
+    {
+        float x = (float)(e.XDelta);    // the move distance in x-axis
+        float y = (float)(e.YDelta);    // the move distance in y-axis
+        // do your action
+    }
+}
+
+private void KeyDownEventHandler(object sender, KeyboardKeyEventArgs e)
+{
+    switch(e.Key)
+    {
+        case Key.Right:                 // press right key
+            // do your action
+            break;
+        case Key.Left:                  // press left key
+            // do your action
+            break;
+        case Key.Up:                    // press up key
+            // do your action
+            break;
+        case Key.Down:                  // press down key
+            // do your action
+            break;
+        case Key.Number0:               // press number 0 key
+            // do your action
+            break;
+        default:
+            break;
+    }
+}
+
+```
+
+## Window related Event
+
+`Window` also provides some window state related events such as `Closed`, `Closing`, `FocusedChanged`, `Resize`, `VisibleChanged`, `WindowStateChanged`, etc. In some scenario, you may want to mirror the state of `Window`, then 
+you can add event handlers on these events.
+
```

Comment:
Please change to:
`Window` also provides some window state related events, such as `Closed`, `Closing`, `FocusedChanged`, `Resize`, `VisibleChanged`, `WindowStateChanged`, and so on. 
In some scenario, if you want to mirror the state of `Window`, then you can add event handlers on these events.

Reply:
@an87li. This comment is fixed, please ignore this comment.